### PR TITLE
[DOCS] Updating system requirements for OpenVINO

### PR DESCRIPTION
**Pull request by sgolebiewski-intel**
Moving info on system requirements for distributions of OpenVINO to one article.

### CODE REVIEW

File: docs/articles_en/about-openvino/release-notes-openvino/system-requirements.rst [hunk 1]
```diff
@@ -162,6 +167,208 @@ Operating systems and developer environment
...
+            :name: archive-lnx-sysreq
+
+            * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`__
+            * `Python 3.9 - 3.12, 64-bit <https://www.python.org/downloads/>`__
+            * GCC:
+
+            .. tab-set::
+
+               .. tab-item:: Ubuntu 20.04
+                  :sync: ubuntu-20
```
Comment (ilya-lavrenov) [inline docs/articles_en/about-openvino/release-notes-openvino/system-requirements.rst:190]: should we also cover Ubuntu 22 and 24 ?

File: docs/articles_en/about-openvino/release-notes-openvino/system-requirements.rst [hunk 2]
```diff
@@ -162,6 +167,208 @@ Operating systems and developer environment
...
+
+            * `CMake 3.13 or higher <https://cmake.org/download/>`__ (choose "macOS 10.13 or later"). Add ``/Applications/CMake.app/Contents/bin`` to path (for default install).
+            * `Python 3.9 - 3.12 <https://www.python.org/downloads/mac-osx/>`__ (choose 3.9 - 3.12). Install and add to path.
+            * Apple Xcode Command Line Tools. In the terminal, run ``xcode-select --install`` from any directory
+            * (Optional) Apple Xcode IDE (not required for OpenVINO™, but useful for development)
+
+         .. tab-item:: Windows
+            :name: archive-win-sysreq
+
+            * `C++ libraries (included in Visual Studio redistributable) <https://aka.ms/vs/17/release/vc_redist.x64.exe>`__
```
Comment (ilya-lavrenov) [inline docs/articles_en/about-openvino/release-notes-openvino/system-requirements.rst:230]: can we also state that it's a dependency of OpenVINO runtime itself?

File: docs/articles_en/about-openvino/release-notes-openvino/system-requirements.rst [hunk 3]
```diff
@@ -162,6 +167,208 @@ Operating systems and developer environment
...
+   .. tab-item:: APT
+      :sync: apt-sysreq
+
+      .. tab-set::
+
+         .. tab-item:: Linux
+            :sync: linux
+
+            * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`__
+            * GCC 7.5.0 (for Ubuntu 18.04), GCC 9.3.0 (for Ubuntu 20.04) or GCC 11.3.0 (for Ubuntu 22.04)
```
Comment (ilya-lavrenov) [inline docs/articles_en/about-openvino/release-notes-openvino/system-requirements.rst:244]: we don't support Ubuntu 18.04 anymore, but let's add Ubuntu 24.04

File: docs/articles_en/about-openvino/release-notes-openvino/system-requirements.rst [hunk 4]
```diff
@@ -162,6 +167,208 @@ Operating systems and developer environment
...
+      :name: homebrew-sysreq
+
+      .. tab-set::
+
+         .. tab-item:: Linux
+            :sync: linux
+
+            * `Homebrew <https://brew.sh/>`_
+            * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`__
+            * GCC 7.5.0 (for Ubuntu 18.04), GCC 9.3.0 (for Ubuntu 20.04) or GCC 11.3.0 (for Ubuntu 22.04)
```
Comment (ilya-lavrenov) [inline docs/articles_en/about-openvino/release-notes-openvino/system-requirements.rst:257]: the same, replace 18.04 with 24.04

File: docs/articles_en/about-openvino/release-notes-openvino/system-requirements.rst [hunk 5]
```diff
@@ -162,6 +167,208 @@ Operating systems and developer environment
...
+            * Oracle Linux 8.7, 8.8 and 9.2-9.4
+            * Fedora 29 and higher up to 41
+            * OpenEuler 20.03, 22.03, 23.03 and 24.03
+            * Anolis OS 8.6 and 8.8
+            * CentOS Stream 8 and 9
+
+            Software:
+
+            * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
+            * GCC 8.2.0
```
Comment (ilya-lavrenov) [inline docs/articles_en/about-openvino/release-notes-openvino/system-requirements.rst:354]: I suppose compiler version should be the same as for archive RHEL (e.g. 8.4.1)

File: docs/articles_en/get-started/install-openvino/install-openvino-archive-windows.rst [hunk 6]
```diff
@@ -33,16 +33,11 @@ System Requirements
...
-      :sync: software
-
-      * `Microsoft Visual Studio 2019 with MSBuild <https://visualstudio.microsoft.com/vs/older-downloads/>`__ or `Microsoft Visual Studio 2022 <http://visualstudio.microsoft.com/  downloads/>`__
-      * `CMake 3.14 or higher, 64-bit <https://cmake.org/download/>`__ (optional, only required for building sample applications)
-      * `Python 3.9 - 3.12, 64-bit <https://www.python.org/downloads/windows/>`__
 
       .. note::
 
-         To install Microsoft Visual Studio 2019, follow the `Microsoft Visual Studio installation guide <https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio?view=vs-2019>`__. You can choose to download the Community version. During installation in the **Workloads** tab, choose **Desktop development with C++**.
+         To install Microsoft Visual Studio, follow the `Microsoft Visual Studio installation guide <https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio?view=vs-2019>`__.
```
Comment (ilya-lavrenov) [inline docs/articles_en/get-started/install-openvino/install-openvino-archive-windows.rst:39]: We can refer to MSVC 2022 version, no needs to stick to 2019

File: docs/articles_en/about-openvino/release-notes-openvino/system-requirements.rst (hunk 1 above)
Comment (sgolebiewski-intel) [reply docs/articles_en/about-openvino/release-notes-openvino/system-requirements.rst]: I have applied your suggestions in:

https://github.com/openvinotoolkit/openvino/pull/27233/commits/f0dcf76a28c43b3943e2687506339e9c585a9a85

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**